Sources/Camille/KarmaBot.swift

```diff
@@ -33,21 +33,24 @@ struct KarmaBotOptions {
     let removeReaction: String?
     
     let textDistanceThreshold: Int
+    let allowedBufferCharacters: [Character]
     
     init(
         targets: [String]? = nil,
         addText: String? = nil,
         addReaction: String? = nil,
         removeText: String? = nil,
         removeReaction: String? = nil,
-        textDistanceThreshold: Int = 4
+        textDistanceThreshold: Int = 4,
+        allowedBufferCharacters: [Character] = [Character(" "), Character(":")]
```

Comment:
new features! it keeps threads alive and you can do smaller code reviews within a PR now too :D